Userland/Applications/PixelPaint/FilterModel.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2021, Tobias Christiansen <tobyase@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "ImageEditor.h"
+#include <AK/NonnullRefPtr.h>
+#include <LibGUI/Model.h>
+
+namespace PixelPaint {
+
+class FilterModel final : public GUI::Model {
+
+public:
+    class FilterInfo : public RefCounted {
+    public:
+        enum Type {
```

Comment:
Perhaps make this an `enum class`? I feel like the code base tends to use those more often.